requirements_all_ds.txt

```diff
@@ -12,22 +12,22 @@ td-client==1.0.0
 pymssql==2.1.4
 dql==0.5.26
 dynamo3==0.4.10
-boto3>=1.10.0,<1.11.0
-botocore>=1.13,<1.14.0
+boto3>=1.21.0,<1.22.0
+botocore>=1.24.0,<1.25.0
 sasl>=0.1.3
 thrift>=0.8.0
 thrift_sasl>=0.1.0
 cassandra-driver==3.21.0
 memsql==3.0.0
 atsd_client==3.0.5
 simple_salesforce==0.74.3
-PyAthena>=1.5.0
+PyAthena>=1.5.0,<=1.11.5
 pymapd==0.19.0
 qds-sdk>=1.9.6
 ibm-db>=2.0.9
 pydruid==0.5.7
 requests_aws_sign==0.1.5
-snowflake-connector-python==2.1.3
+snowflake-connector-python==2.7.4
```

Comment:
Why bump the snowflake connector in this PR?

Reply:
Docker build succeeds via 2.1.3, but shows message.

```
snowflake-connector-python 2.1.3 requires boto3<1.11.0,>=1.4.4, but you'll have boto3 1.21.46 which is incompatible.
snowflake-connector-python 2.1.3 requires botocore<1.14.0,>=1.5.0, but you'll have botocore 1.24.46 which is incompatible.
snowflake-connector-python 2.1.3 requires certifi<2021.0.0, but you'll have certifi 2022.6.15 which is incompatible.
snowflake-connector-python 2.1.3 requires pytz<2021.0, but you'll have pytz 2022.1 which is incompatible.
snowflake-connector-python 2.1.3 requires requests<2.23.0, but you'll have requests 2.27.1 which is incompatible.
snowflake-connector-python 2.1.3 requires urllib3<1.26.0,>=1.20, but you'll have urllib3 1.26.10 which is incompatible.
```

snowflake-connector-python==2.1.3 depens on botocore and requires old version.
After ver snowflake-connector-python==2.7.0 removed these dependencies completely.

I read the source code and release notes of snowflake-connector-python, but there are no breaking changes, and the implementation does not depend on botocore.
I think it's better to update to a version that doesn't depend on botocore unless you have a specific reason.
(The maintainer of snowflake-connector-python seems to be an engineer of the snowflake.)

Reply:
Thanks for that explanation. I agree with your proposal 👌